app/Http/Controllers/LandingPagesController.php

```diff
@@ -42,6 +42,11 @@ public function show($id, $slug = null)
             return $canonicalRedirect;
         }
 
+        if ($item->is_unlisted) {
```

Comment:
I'm not sure the LandingPage model has an `is_unlisted` attribute, does it? If not, you may want to use the `published` property:

```
if ($item->is_published) {
```

Reply:
@nikhiltri we'd want to check if it isn't published and then set the seo properties to no follow and no index, right? so it'd be if (!$item->is_published) {

Reply:
Yes!